.ci/config/matrices.yaml

```diff
@@ -32,8 +33,8 @@ engine-rest:
 daily-stages:
   databases:
 # daily-stages.databases should include the latest version for each supported database
-  - 'postgresql_160'
```

Comment:
**Context:** The postgres & aurora stages are removed for faster testing.
**When:** Testing is complete
**Action:** Uncomment-out postgresstages

Reply:
Added the entry back during merge with latest Support for AWS Aurora Postgres 16 from `master`.